cms/djangoapps/contentstore/tests/test_contentstore.py

```diff
@@ -1746,6 +1746,13 @@ def test_rerun_course_success(self):
         self.assertEqual(1, len(source_videos))
         self.assertEqual(source_videos, target_videos)
 
+    def test_rerun_course_resets_advertised_date(self):
```

Comment:
@sarina I naively assumed every function should have a doc string. I'm cool not having one in this case, as I agree that it's abundantly clear what the test is doing